BaseFilter/sections/foreign.txt

```diff
@@ -5476,8 +5476,12 @@ vg.no###ad-topboard
 !---------------------------------------------------------------------
 ! NOTE: Persian
 !
-! https://github.com/AdguardTeam/AdguardFilters/issues/146668
-||cdn.hostdl.com/assets/soft98.gif$image,redirect=1x1-transparent.gif
+! https://github.com/AdguardTeam/AdguardFilters/issues/152739
+!#if (adguard_app_windows || adguard_app_mac || adguard_app_android || adguard_ext_firefox)
+||soft98.ir^$script,~third-party,replace=/clipPath/filter/
```

Comment:
As I can see, this rule only applies to one script. Do you have the same?
If so, it's better to do a more specific rule like this.
```adblock
||soft98.ir/templates/*/js/jquery.min.packed.js$replace=/clipPath/filter/
```